javagen/src/main/java/com/azure/autorest/mapper/ProxyParameterMapper.java

```diff
@@ -58,6 +58,10 @@ public ProxyMethodParameter map(Parameter parameter) {
 
         Schema parameterJvWireType = parameter.getSchema();
         IType wireType = Mappers.getSchemaMapper().map(parameterJvWireType);
+        // fluent premium treats subscriptionId as string in order not to break ManagementClients and client methods
+        if (settings.isFluentPremium() && "subscriptionId".equals(CodeNamer.getPropertyName(name))) {
```

Comment:
why `isFluentPremium`?

Reply:
Thought we may break existing managementClient in lite?

Reply:
It does not break. Officially ClientManagement is not included in revapi.

Reply:
OK. FluentTransformer should be a better place to make the change then.